src/test/java/net/starlark/java/eval/BUILD

```diff
@@ -66,6 +63,25 @@ java_test(
     ],
 )
 
+# Script-based tests of the Starlark interpreter.
+java_test(
+    name = "ScriptTest",
+    use_testrunner = False,
+    runtime_deps = [
+        ":ScriptTest_lib",
+    ],
+)
+
+java_test(
+    name = "ScriptTest_Latin1",
```

Comment:
I'm slightly worried about redundantly rerunning non-string-related tests. Can we filter this down to just those test suites that actually are affected by encoding/string behavior?

Reply:
The test passes in 0.8s on my machine. I could explicitly list all tests that are affected at this point in time, but I'm pretty certain that I would miss some - not even taking into account that the list is bound to become stale over time. Given how subtle the difference is, I have a strong preference for just running all tests twice.